plugins/org.eclipse.osee.orcs.rest/src/org/eclipse/osee/orcs/rest/internal/RelationEndpointImpl.java

```diff
@@ -91,6 +93,31 @@ public List<RelationTypeToken> convertRelations(ArtifactId artToken, RelationTyp
       return null;
    }
    
+   @Override
+   public List<RelationTypeToken> convertAllRelations(RelationTypeToken oldRelationType,
+      RelationTypeToken newRelationType) {
+      orcsApi.tokenService().registerRelationType(ShadowCoreRelationTypes.DefaultHierarchicalRel);
+      RelationTypeSide sideB = new RelationTypeSide(oldRelationType, RelationSide.SIDE_B);
+      TransactionBuilder tx = orcsApi.getTransactionFactory().createTransaction(branch,
+         String.format("Converting relations for Old Relation = %s New Relation = %s",
+            oldRelationType.getName(), oldRelationType.getName()));
+      
+      for (ArtifactReadable art : orcsApi.getQueryFactory().fromBranch(branch).andRelationExists(oldRelationType).asArtifacts()) {
+         
+            if (art.getRelatedCount(sideB) > 0) {
+            
+            for (ArtifactReadable artB : art.getRelated(sideB).getList()) {
```

Comment:
I think we need logic for preserving the order here...

Reply:
I think this does return in order, but I'll validate that.

Reply:
no...what I'm saying is that the user might have defined a custom order(i.e. on the attribute), and we need to handle that here and put the order in the relation.

Reply:
ok, had to switch to using getResults